data-sets-api-server/src/test/java/org/zowe/unix/files/controller/UnixFilesControllerTest.java

```diff
@@ -80,13 +79,13 @@ public void get_directory_listing_success() throws Exception {
             .size(8192).lastModified("2019-02-03T16:04:19").children(children).build();
         String path = "/u/ibmuser";
 
-        when(unixFilesService.listUnixDirectory(path)).thenReturn(listedDirectory);
+        when(unixFilesService.listUnixDirectory(path, "http://localhost/api/v1/unixfiles")).thenReturn(listedDirectory);
```

Comment:
Maybe consider extracting "http://localhost/api/v1/unixfiles" to a constant incase we want to update/use it elsewhere?